### DISCUSSION TITLE
gush

### DISCUSSION THREAD

**Discussion by omegajudith** [Cyfrin/foundry-full-course-cu#2793] [General]
everything perfect, learn is perfect and everything just wow, but gets so frustrating when i have to find the source code on GitHub. duhhhhhhhhhh?

**Comment by EngrPips** [comment]
Hello @omegajudith, Do you have any problem finding the source code on Github.

**Reply by omegajudith** [reply]
Yes so hard. There's hundreds of folders and links and when u open you don't get the particular topics your studying. Sometimes I see Patrick opening code and I don't know where he got it so I assume it got from the repository but he finding it is wild

**Reply by EngrPips** [reply]
You can just go to the `code` tab of this GitHub repository and go to the section for which you want to get its code there is always a the GitHub link to the code for each section under the section.

<img width="1440" alt="Screenshot 2024-11-01 at 20 17 10" src="https://github.com/user-attachments/assets/6677e785-09a7-4b70-b8c7-22a05810189b">
